db/models/plan.js

```diff
@@ -1,11 +1,31 @@
-const Sequelize = require('sequelize')
-const db = require('../db')
+const Sequelize = require('sequelize');
+const db = require('../db');
 
-const ModelName = db.define('plan', {
+const Plan = db.define('plan', {
+  duration: {
+    type: Sequelize.INTEGER,
+    defaultValue : 1
+  },
+  end_date : {
+    type : Sequelize.DATE,
+    allowNull: true
+  },
+  underscored : true
```

Comment:
Needs to be moved into Options object. (Can be placed before 'instanceMethods' at line 16.)